api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -812,16 +783,22 @@ public static int compareWithNullAsLatest(Date d1, Date d2) {
 		return ret;
 	}
 	
-	// TODO: properly handle duplicates
 	public static List<Concept> conceptListHelper(String descriptor) {
 		List<Concept> ret = new ArrayList<>();
+		Set<String> duplicates = new HashSet<>();
```

Comment:
I believe implementing the TODO is also beyond the scope of this issue

Reply:
Ok. I will revert